docs/Configuration/yaml-files.md

```diff
@@ -571,12 +571,11 @@ org_settings:
 At the team level, there is the additional option to enable conditional access, which blocks third party app sign-ins on hosts failing policies. (Available in Fleet Premium for managed cloud customers. Must have Microsoft Entra connected.)
 
 ```yaml
-integrations:
-  conditional_access_enabled: true
+team_settings:
+  integrations:
+    conditional_access_enabled: true
```

Comment:
See https://github.com/fleetdm/fleet/blob/82a5b80da698164d0cb672629afe1e6bf1ed1075/it-and-security/teams/workstations.yml#L11.